packages/dds/sequence/src/revertibles.ts

```diff
@@ -323,7 +323,9 @@ export function appendSharedStringDeltaToRevertibles(
 				event: IntervalOpType.POSITION_REMOVE,
 				intervals: [],
 				revertibleRefs,
-				mergeTreeRevertible: removeRevertibles[0],
+				// TODO Non null asserting, why is this not null?
```

Comment:
We just asserted that the length of `removeRevertibles` was 1, so you can update the comment to reflect that.